README.md

```diff
@@ -6,14 +6,40 @@
 ![version](https://img.shields.io/github/v/release/keptn/lifecycle-toolkit)
 [![GitHub Discussions](https://img.shields.io/github/discussions/keptn/lifecycle-toolkit)](https://github.com/keptn/lifecycle-toolkit/discussions)
 
-The Keptn Lifecycle Toolkit (KLT) provides a “cloud-native” approach
+This is the primary repository for
+the Keptn Lifecycle Toolkit (KLT) software and documentation.
+KLT provides a “cloud-native” approach
 for managing the application release lifecycle
+metrics, observability, health checks,
 with pre- and post-deployment evaluations and tasks.
-It also supports application health checks,
-metrics, and observability.
 It is an incubating project, under the umbrella of the
 [Keptn Application Lifecycle working group](https://github.com/keptn/wg-app-lifecycle).
 
+> **Note**
+  The primary repository for Keptn v1 is
+  [https://github.com/keptn/keptn](https://github.com/keptn/keptn)
```

Comment:
Please remove that note completely for now :)

Reply:
yes, since we announce Keptn v1 EOL - this is the main repository

Reply:
Done, although I do think people are still confused about the two products.